openrtb_ext/imp_seedingAlliance.go

```diff
@@ -2,4 +2,5 @@ package openrtb_ext
 
 type ImpExtSeedingAlliance struct {
 	AdUnitID string `json:"adUnitID"`
+	SeatID   string `json:"seatID"`
```

Comment:
static/bidder-params/seedingAlliance.json specifiesincomming request will have `seatId`. Therefore should update struct to have `seatId` in json tag 

```
type ImpExtSeedingAlliance struct {
	AdUnitID string `json:"adUnitID"`
	SeatID   string `json:"seatId"`
}
```

or change static/bidder-params/seedingAlliance.json to use `seatID`

Reply:
also should update param tests for to test seatID param

https://github.com/prebid/prebid-server/blob/ec729e6e5d50a1306d76332fda52108b9d313c1d/adapters/seedingAlliance/params_test.go#L36-L43